scipy/stats/_morestats.py

```diff
@@ -4061,26 +4075,37 @@ def directionalmean(samples, *, axis=0, normalize=True):
 
     .. code-block:: python
 
-        mean=samples.mean(axis=0)
-        directionalmean = mean/np.linalg.norm(mean)
+        mean = samples.mean(axis=0)
+        magnitude = np.linalg.norm(mean)
+        directional_mean = mean / magnitude
 
     This definition is appropriate for *directional* data (i.e. vector data
     for which the magnitude of each observation is irrelevant) but not
     for *axial* data (i.e. vector data for which the magnitude and *sign* of
     each observation is irrelevant).
 
+    The function also returns the magnitude of mean resultant vector which
+    can be used to calculate directional variance. Several definitions of
+    directional variance have been proposed e.g. ``Var(z) = 1 - R`` (where
```

Comment:
I'd include the 2(1-R) definition, too, because that stood out to me as the standard, probably because it was the only one explicitly called the "spherical variance".

Reply:
Done.